DEVELOPMENT.md

```diff
@@ -258,6 +259,16 @@ Environment variables
 TEST_WORK is used by registry server unit-test code to specify path to certificates in test system. 
 Note, THIS IS NOT USED IN PRODUCTION
 
+Logging
+-------
+
+The klog.Info statements are used via the verbosity checker using the following levels:
```

Comment:
Are these levels derived from the k8s klog logging conventions doc at https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md ?
Looking at the two, I'm not sure they align? If they did, then it would probably makes sense to add a reference here.

Reply:
Our levels developed separately and are about "off by one" related to guidelines pointed by this link, that's actually documented in issue this PR is addressing.
We discussed that off-line and current understanding is, we do not want to shift all V-levels right now in pmem-csi code.

Reply:
I was thinking about adding reference, but then the obvious question will be "hey why are you off by one?", and right now the idea is we may want to make some other changes to logging, so it's not worth it to change all the Info code lines (plus all deployment files which set the verbosity level).